docs/site-example/i18n/en.po

```diff
@@ -32,10 +32,10 @@ msgstr ""
 
 msgid "gluon-config-mode:reboot"
 msgstr ""
-"<p>The node is currently rebooting and will try to connect to other "
-"nearby Freifunk nodes after that. "
+"<p>Your node <em><%= escape(hostname) %></em> is currently rebooting and will "
+"try to connect to other nearby Freifunk nodes after that. "
 "For more information on the Freifunk Duckburg community, have a look at "
-"<a href=\"https://entenhausen.freifunk.net/\">our homepage</a>.</p>"
+"<a href=\"https://entenhausen.freifunk.net/#<%= sysconfig.primary_mac %>\">our homepage</a>.</p>"
```

Comment:
These changes don't make sense. Please just don't change the examples at all, at least not in this PR.